From db4bcbd5ca4f22dda263a15aa485959878004a58 Mon Sep 17 00:00:00 2001
From: Valentine Barshak <valentine.barshak@cogentembedded.com>
Date: Tue, 26 Nov 2019 21:52:37 +0300
Subject: [PATCH 22/22] arm: dts: renesas: Disable RPC on all boards

This disables RPC device on all R-Car V3x, Salvator-X, and ULCB boards.

Signed-off-by: Valentine Barshak <valentine.barshak@cogentembedded.com>
---
 arch/arm/dts/r8a77950-salvator-x-u-boot.dts | 2 +-
 arch/arm/dts/r8a77950-ulcb-u-boot.dts       | 2 +-
 arch/arm/dts/r8a77960-salvator-x-u-boot.dts | 2 +-
 arch/arm/dts/r8a77960-ulcb-u-boot.dts       | 2 +-
 arch/arm/dts/r8a77965-salvator-x-u-boot.dts | 2 +-
 arch/arm/dts/r8a77965-ulcb-u-boot.dts       | 2 +-
 arch/arm/dts/r8a77970-eagle-u-boot.dts      | 4 ++--
 arch/arm/dts/r8a77970-v3msk-u-boot.dts      | 4 ++--
 arch/arm/dts/r8a77980-condor-u-boot.dts     | 4 ++--
 arch/arm/dts/r8a77980-v3hsk-u-boot.dts      | 4 ++--
 10 files changed, 14 insertions(+), 14 deletions(-)

diff --git a/arch/arm/dts/r8a77950-salvator-x-u-boot.dts b/arch/arm/dts/r8a77950-salvator-x-u-boot.dts
index b1d3a7c..967fa3b 100644
--- a/arch/arm/dts/r8a77950-salvator-x-u-boot.dts
+++ b/arch/arm/dts/r8a77950-salvator-x-u-boot.dts
@@ -38,5 +38,5 @@
 };
 
 &rpc {
-	status = "okay";
+	status = "disabled";
 };
diff --git a/arch/arm/dts/r8a77950-ulcb-u-boot.dts b/arch/arm/dts/r8a77950-ulcb-u-boot.dts
index 4dd4f4e..30897a6 100644
--- a/arch/arm/dts/r8a77950-ulcb-u-boot.dts
+++ b/arch/arm/dts/r8a77950-ulcb-u-boot.dts
@@ -38,5 +38,5 @@
 };
 
 &rpc {
-	status = "okay";
+	status = "disabled";
 };
diff --git a/arch/arm/dts/r8a77960-salvator-x-u-boot.dts b/arch/arm/dts/r8a77960-salvator-x-u-boot.dts
index 4c4fc6f..536df64 100644
--- a/arch/arm/dts/r8a77960-salvator-x-u-boot.dts
+++ b/arch/arm/dts/r8a77960-salvator-x-u-boot.dts
@@ -38,5 +38,5 @@
 };
 
 &rpc {
-	status = "okay";
+	status = "disabled";
 };
diff --git a/arch/arm/dts/r8a77960-ulcb-u-boot.dts b/arch/arm/dts/r8a77960-ulcb-u-boot.dts
index 40e0b4a..b97f31c 100644
--- a/arch/arm/dts/r8a77960-ulcb-u-boot.dts
+++ b/arch/arm/dts/r8a77960-ulcb-u-boot.dts
@@ -38,5 +38,5 @@
 };
 
 &rpc {
-	status = "okay";
+	status = "disabled";
 };
diff --git a/arch/arm/dts/r8a77965-salvator-x-u-boot.dts b/arch/arm/dts/r8a77965-salvator-x-u-boot.dts
index eea7911..032c905 100644
--- a/arch/arm/dts/r8a77965-salvator-x-u-boot.dts
+++ b/arch/arm/dts/r8a77965-salvator-x-u-boot.dts
@@ -41,5 +41,5 @@
 };
 
 &rpc {
-	status = "okay";
+	status = "disabled";
 };
diff --git a/arch/arm/dts/r8a77965-ulcb-u-boot.dts b/arch/arm/dts/r8a77965-ulcb-u-boot.dts
index b1cddb1..3548f49 100644
--- a/arch/arm/dts/r8a77965-ulcb-u-boot.dts
+++ b/arch/arm/dts/r8a77965-ulcb-u-boot.dts
@@ -40,5 +40,5 @@
 };
 
 &rpc {
-	status = "okay";
+	status = "disabled";
 };
diff --git a/arch/arm/dts/r8a77970-eagle-u-boot.dts b/arch/arm/dts/r8a77970-eagle-u-boot.dts
index eb868ed..63b9c44 100644
--- a/arch/arm/dts/r8a77970-eagle-u-boot.dts
+++ b/arch/arm/dts/r8a77970-eagle-u-boot.dts
@@ -36,7 +36,7 @@
 
 &rpc {
 	num-cs = <1>;
-	status = "okay";
+	status = "disabled";
 	spi-max-frequency = <50000000>;
 	#address-cells = <1>;
 	#size-cells = <0>;
@@ -49,6 +49,6 @@
 		spi-tx-bus-width = <1>;
 		spi-rx-bus-width = <1>;
 		reg = <0>;
-		status = "okay";
+		status = "disabled";
 	};
 };
diff --git a/arch/arm/dts/r8a77970-v3msk-u-boot.dts b/arch/arm/dts/r8a77970-v3msk-u-boot.dts
index 6ee06d7..5ae02ae 100644
--- a/arch/arm/dts/r8a77970-v3msk-u-boot.dts
+++ b/arch/arm/dts/r8a77970-v3msk-u-boot.dts
@@ -47,7 +47,7 @@
 
 &rpc {
 	num-cs = <1>;
-	status = "okay";
+	status = "disabled";
 	spi-max-frequency = <50000000>;
 	#address-cells = <1>;
 	#size-cells = <0>;
@@ -60,6 +60,6 @@
 		spi-tx-bus-width = <1>;
 		spi-rx-bus-width = <1>;
 		reg = <0>;
-		status = "okay";
+		status = "disabled";
 	};
 };
diff --git a/arch/arm/dts/r8a77980-condor-u-boot.dts b/arch/arm/dts/r8a77980-condor-u-boot.dts
index 1b22c7f..68ebad7 100644
--- a/arch/arm/dts/r8a77980-condor-u-boot.dts
+++ b/arch/arm/dts/r8a77980-condor-u-boot.dts
@@ -16,7 +16,7 @@
 
 &rpc {
 	num-cs = <1>;
-	status = "okay";
+	status = "disabled";
 	spi-max-frequency = <50000000>;
 	#address-cells = <1>;
 	#size-cells = <0>;
@@ -29,6 +29,6 @@
 		spi-tx-bus-width = <1>;
 		spi-rx-bus-width = <1>;
 		reg = <0>;
-		status = "okay";
+		status = "disabled";
 	};
 };
diff --git a/arch/arm/dts/r8a77980-v3hsk-u-boot.dts b/arch/arm/dts/r8a77980-v3hsk-u-boot.dts
index d083df6..4842314 100644
--- a/arch/arm/dts/r8a77980-v3hsk-u-boot.dts
+++ b/arch/arm/dts/r8a77980-v3hsk-u-boot.dts
@@ -16,7 +16,7 @@
 
 &rpc {
 	num-cs = <1>;
-	status = "okay";
+	status = "disabled";
 	spi-max-frequency = <50000000>;
 	#address-cells = <1>;
 	#size-cells = <0>;
@@ -29,7 +29,7 @@
 		spi-tx-bus-width = <1>;
 		spi-rx-bus-width = <1>;
 		reg = <0>;
-		status = "okay";
+		status = "disabled";
 	};
 };
 
-- 
2.7.4

